docs/metrics-analysis-prometheus/prometheus.md

```diff
@@ -2,8 +2,8 @@
 
 ## Introduction
 
-Prometheus is integrated with our `gh_actions_exporter` repository,
-enabling the export of GitHub Actions metrics.
+In order to collect and analyze GitHub Actions metrics, users are expected
+to have an existing Prometheus installation and configure it to pull metrics.
 
 Prometheus is a powerful open-source monitoring and alerting system that allows
```

Comment:
I feel like those three lines deserve to be first sentence of the introduction. No?

Reply:
True